homeassistant/components/airzone/select.py

```diff
@@ -95,7 +130,21 @@ def _async_entity_listener() -> None:
         received_zones = set(zones_data)
         new_zones = received_zones - added_zones
         if new_zones:
-            async_add_entities(
+            entities: list[AirzoneZoneSelect] = []
+            entities += [
```

Comment:
```suggestion
            entities: list[AirzoneZoneSelect] = [
```

Reply:
Done in 38e6f6b57b18eae688f19f737c13f377547b4539